### DISCUSSION TITLE
Very quick question about migrating to pnpm

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [pnpm/pnpm#5460] [Q&A]
1. Do we need to remove all packages installed globally with npm, and install them with pnpm, 
so we can use pnpm from now on, and avoid conflicts?

2. How do we know when to use --shamefully-hoist flag and when not?

Kind replies will be read and very appreciated.

**Comment by zkochan** [comment]
Packages installed globally you mean with `npm install --global <pkg>`? It is your choice.

> How do we know when to use --shamefully-hoist flag and when not?

If something doesn't work but it worked before, you can try `shamefully-hoist=true`. If that doesn't help, use `node-linker=hoisted`.
